src/components/BookingModal/BookingForm/index.js

```diff
@@ -0,0 +1,168 @@
+import React, { Fragment } from 'react';
+import { PropTypes as PT } from 'prop-types';
+import container from './container';
+import { Form, Input } from '../../common';
+import eventTypes from '../../../utilities/eventTypes';
+import holidayStatus from '../../../utilities/holidayStatus';
+
+const BookingForm = props => {
+  const {
+    isEventBeingUpdated,
+    createEvent,
+    updateEvent,
+    formData,
+    formStatus,
+    formIsValid,
+    bookingDuration,
+    booking,
+    workingFromHomeBooking,
+    submitButtonDisabled,
+    availableDays,
+  } = props;
+
+  const {
+    eventStatus: { eventStatusId },
+  } = booking;
+  const isEventCancelled = eventStatusId === holidayStatus.CANCELLED;
+
+  const { eventTypeId } = formData;
+
+  const createCtas = () => {
```

Comment:
This is fine how it is, but the idea behind the container/component pattern is that all logic is handled in the container, and the component is solely used for displaying data.

Reply:
I may be asking a silly question but what is `createCtas`? i get the `create` part but not the `Ctas`

Reply:
I'm guessing it means "call to action"s - some element that encourages interaction from the user; in this case buttons @lee-cheung